client/additional-methods-setup/upe-preview-methods-selector/add-payment-methods-task.js

```diff
@@ -171,14 +171,14 @@ const AddPaymentMethodsTask = () => {
 							'woocommerce-payments'
 						),
 						components: {
-							// TODO
 							learnMoreLink: (
-								<a href="?TODO">
+								// eslint-disable-next-line max-len
+								<ExternalLink href="https://docs.woocommerce.com/document/payments/additional-payment-methods/#available-methods">
```

Comment:
Changed to be an external link
![Markup 2021-07-21 at 15 55 39](https://user-images.githubusercontent.com/273592/126566157-3ba1785e-8363-469b-a057-0c0cf5b0b66b.png)